src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -259,5 +261,68 @@ private static CompletionItemRules MakeRule(bool importDirective, bool preselect
         }
 
         protected override CompletionItemSelectionBehavior PreselectedItemSelectionBehavior => CompletionItemSelectionBehavior.HardSelection;
+
+        protected override CompletionItem CreateItem(
+            CompletionContext completionContext,
+            string displayText,
+            string displayTextSuffix,
+            string insertionText,
+            List<ISymbol> symbols,
+            SyntaxContext context,
+            bool preselect,
+            SupportedPlatformData supportedPlatformData)
+        {
+            var rules = CreateCompletionItemRules(completionContext, symbols, context, preselect);
+            var symbol = symbols[0];
+            var item = SymbolCompletionItem.CreateWithNameAndKind(
```

Comment:
Can't we call `base.CreateItem` instead, and add the additional property to the returned item?